File: app/src/main/java/com/neome/java/api/ent/base/dto/DtoEntUserAvatar.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.base.dto;

import com.neome.java.api.meta.base.Types.EntUserId;
import com.neome.java.api.meta.base.Types.MediaId;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DtoEntUserAvatar
{
  @Nullable
  public MediaId avatarId;

  public EntUserId entUserId;

  public String handle;

  public String nickName;
}

File: app/src/main/java/com/neome/api/ent/base/dto/DtoEntUserAvatar.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.base.dto

import com.neome.api.meta.base.Types.EntUserId
import com.neome.api.meta.base.Types.MediaId

interface DtoEntUserAvatar
{
  val avatarId: MediaId?
  val entUserId: EntUserId
  val handle: String
  val nickName: String
}